### DISCUSSION TITLE
Currently recommended version

### DISCUSSION THREAD

**Discussion by jacksonludwig** [storybookjs/react-native#387] [Q&A] [answered]
I can see that version 6.0 has been in development for quite some time. If I am starting a new react native storybook project, should I still be using 5.3?

**Comment by dannyhw** [comment, marked as answer]
TLDR: v6 beta in my (biased) opinion, though it depends on your usecase.

@jacksonludwig hey jackson, I guess its all down to your needs. If you need a stable version then you can feel free to pick 5.3. No changes have been made to 5.3 for some time but it still works.

Consider that the package @storybook/react-native-server won't be supported initially for v6 so if you need that feature (controlling the device remotely) then it might make sense to stick with v5 for now.

If you want the new features of storybook like CSF, args, controls and auto imports then you can try the beta. 

Considering that for many Storybook is a dev tool only I don't think there is a big risk to using the beta version since it won't effect your end user.

Also by starting with 5.3 then when v6 comes out there will be some work to migrate if/when you want to upgrade.

Recently I was able to get past the hurdle of getting the new StoryStore introduced in storybook 6.4 working and so once that has been ironed out I don't think it will be much longer before we can make a v6 stable release. Though I've said that before and ran into some delays so take it with a grain of salt 😅 .

**Reply by jacksonludwig** [reply]
Thanks for the detailed break down.
